schemas/1.0/general/decision.json

```diff
@@ -47,9 +47,7 @@
               "type": "string",
               "enum": [
```

Comment:
Should we update the decision struct as well to use the correct enums?

Reply:
Thanks @EdwinKruglov. The decision struct uses Types::FundingDecisionResult which is updated here: https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/pull/219/files#diff-a0939f79f4e37686df77707ace12fbd73e929dd2df816902ebf967f2e243e99aR300

As for the Means/IoJ test result structs, I think its fine for the structs to be looser than the json-schema. We could have a separate Struct for IoJ v Means tests but I don't think it is worth it.